segments/helpers.py

```diff
@@ -136,16 +136,16 @@ def refresh_segment(self, segment_id, sql):
 
     def delete_segment(self, segment_id):
         segment_key = self.segment_key % segment_id
-        for user_id in self.redis.sscan_iter(segment_key):
-            self.remove_segment_membership(segment_id, user_id)
+        cursor = '0'
```

Comment:
This seems a bit hacky, we can probably find a better way to break out of the while loop

Reply:
@apolisskaya That's exactly how they do it in `sscan_iter`